Add documentation for Day 2-7 Activation, Google Play Store and Apple App Store (#545)
mozilla/firefox-data-docs: src/datasets/non_desktop/day_2_7_activation/reference.md

> # Non Desktop Day 2 - 7 Activation  

Reviewer: ```suggestion
# Non-Desktop Day 2-7 Activation  
```

<!-- toc -->

# Introduction

> `firefox_nondesktop_day_2_7_activation` is designed for use in calculating the day 2-7 activation metric, a key result in 2020 for the nondesktop products.

Reviewer: ```suggestion
`firefox_nondesktop_day_2_7_activation` is designed for use in calculating the day 2-7 activation metric, a key result in 2020 for mobile (non-desktop) products.
```

> 

Reviewer: ```suggestion
```

# Contents

> `firefox_nondesktop_day_2_7_activation` is an aggregate table with two key metrics: `new_profiles` and `day_2_7_activated`. These are primarily aggregated by `submission_date`.

Reviewer: We should probably mention the table this is derived from: `nondesktop_clients_last_seen` (we don't have a reference to the non-desktop clients last seen dataset, so let's just reference @jklukas's cookbook which at least mentions it)

```suggestion
`firefox_nondesktop_day_2_7_activation` is a table with two key metrics: `new_profiles` and `day_2_7_activated`, aggregated over `submission_date`. It is derived from the [non-desktop clients last seen table](https://docs.telemetry.mozilla.org/cookbooks/clients_last_seen_bits.html).
```

- `new_profiles`: Unique count of client ids with a given profile creation date. As not all initial pings are received exactly on the day of profile creation, we wait for 7 days after the profile creation date before establishing the New Profile cohort to ensure the data is complete.
- `days_2_7_activated`: Unique count of client ids who use the product at any point starting the day after they created a profile up to 6 days after.

> Due to the delay in receiving all the initial pings and subsequent 7 day wait prior to establishing the new profile cohort, the table will lag other daily usage nondesktop tables e.g. `nondesktop_clients_last_seen` by 7 days. 

Reviewer: I'd suggest maybe putting this under "Background & Caveats"?

```suggestion
```

We also include a variety of dimension information (e.g. `product`, `app_name`, `app_version`, `os`, `normalized_channel` and `country`) to aggregate on. 

> 

Reviewer: ```suggestion

This dataset is backfilled through 2017-01-01.
```

# Accessing the Data
Access the data at [`moz-fx-data-shared-prod.telemetry.firefox_nondesktop_day_2_7_activation`](https://console.cloud.google.com/bigquery?project=moz-fx-data-shared-prod&p=moz-fx-data-shared-prod&d=telemetry&t=firefox_nondesktop_day_2_7_activation&page=table)

# Data Reference
## Example Queries

> [This activation query](https://sql.telemetry.mozilla.org/queries/72054/source) gives the `day 2-7 activation` by product. A new profile is considered activated if the client uses the browser at any point starting 1 day up to 6 days after the `profile creation date`.

Reviewer: Let's put the SQL inline:

```suggestion

This query gives the `day 2-7 activation` by product:

```sql
SELECT
  cohort_date,
  product,
  SUM(day_2_7_activated) as day_2_7_activated,
  SUM(new_profiles) as new_profiles,
  SAFE_DIVIDE(SUM(day_2_7_activated), SUM(new_profiles)) as day_2_7_activation
FROM
  `moz-fx-data-derived-datasets.telemetry.firefox_nondesktop_day_2_7_activation`
WHERE
  cohort_date = "2020-03-01"
GROUP BY 1,2
ORDER BY 1
```
[link](https://sql.telemetry.mozilla.org/queries/72054/source)

A new profile is considered activated if the client uses the browser at any point starting 1 day up to 6 days after the `profile creation date`.
```

## Scheduling
This dataset is scheduled on Airflow ([source](https://github.com/mozilla/telemetry-airflow/blob/59effc6ead0b764a9ef3d30f40fbdb4b0b3394ec/dags/copy_deduplicate.py#L337)).

## Schema

> As of 2020-07-24, the current version of firefox_nondesktop_day_2_7_activation is v1, and has a schema as follows. It's backfilled through 2017-01-01

Reviewer: The backfilled part is probably worth mentioning earlier

```suggestion
As of 2020-07-24, the current version of firefox_nondesktop_day_2_7_activation is v1, and has a schema as follows:
```

```
root
    |- submission_date: date
    |- product: string
    |- app_name: string
    |- app_version: string
    |- os: string
    |- normalized_channel: string
    |- country: string
    |- new_profiles: integer
    |- day_2_7_activated: integer
```

# Code Reference
The firefox_nondesktop_day_2_7_activation job is [defined in `bigquery-etl`](https://github.com/mozilla/bigquery-etl/blob/master/sql/telemetry_derived/firefox_nondesktop_day_2_7_activation_v1/query.sql).

# Background and Caveats

> 

Reviewer: ```suggestion

Due to the delay in receiving all the initial pings and subsequent 7 day wait prior to establishing the new profile cohort, the table will lag `nondesktop_clients_last_seen` by 7 days.

```

- `days_2_7_activated`: Unique count of client ids who use the product at any point starting the day after they created a profile up to 6 days after.

We also include a variety of dimension information (e.g. `product`, `app_name`, `app_version`, `os`, `normalized_channel` and `country`) to aggregate on. 

This dataset is backfilled through 2017-01-01.
# Accessing the Data
Access the data at [`moz-fx-data-shared-prod.telemetry.firefox_nondesktop_day_2_7_activation`](https://console.cloud.google.com/bigquery?project=moz-fx-data-shared-prod&p=moz-fx-data-shared-prod&d=telemetry&t=firefox_nondesktop_day_2_7_activation&page=table)

# Data Reference
## Example Queries

This query gives the `day 2-7 activation` by product:

```sql
SELECT
  cohort_date,
  product,
  SUM(day_2_7_activated) as day_2_7_activated,
  SUM(new_profiles) as new_profiles,
  SAFE_DIVIDE(SUM(day_2_7_activated), SUM(new_profiles)) as day_2_7_activation
FROM
  `moz-fx-data-derived-datasets.telemetry.firefox_nondesktop_day_2_7_activation`
WHERE
  cohort_date = "2020-03-01"
GROUP BY 1,2
ORDER BY 1
```

> [link](https://sql.telemetry.mozilla.org/queries/72054)

Reviewer 2: I think `Link to query in Redash` would be better

Author: updated it

# Non-Desktop Day 2-7 Activation  

<!-- toc -->

# Introduction

> `firefox_nondesktop_day_2_7_activation` is designed for use in calculating the [Day 2-7 Activation metric](../../metrics/metrics.md#day-2-7-activation), a key result in 2020 for non-desktop products.

Reviewer: ```suggestion
`firefox_nondesktop_day_2_7_activation` is designed for use in calculating the [Day 2-7 Activation metric](../../../metrics/metrics.md#day-2-7-activation), a key result in 2020 for non-desktop products.
```
